hadoop-ozone/dist/src/main/compose/upgrade/compose/non-ha/docker-config

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+OZONE-SITE.XML_ozone.metadata.dirs=/data/metadata
+
+OZONE-SITE.XML_ozone.client.failover.max.attempts=6
+
+OZONE-SITE.XML_ozone.om.service.ids=omservice
+OZONE-SITE.XML_ozone.om.nodes.omservice=om1
+OZONE-SITE.XML_ozone.om.address.omservice.om1=om
```

Comment:
Having to add OM service ID for `non-ha` test to pass means one of:

(1) Users with non-HA cluster will need to make config changes for the upgrade.
(2) The test is not modeling real-life upgrade steps correctly.  We need to tweak the test, not the environment.

I tend to think it's the latter.

Proposed test fix: https://github.com/apache/ozone/compare/master...adoroszlai:ozone:refs/heads/HDDS-12327
test run: https://github.com/adoroszlai/ozone/actions/runs/14060297062/job/39370001881

Please take a look.  If you agree, I'll push my additional commits to this PR's branch.

Reply:
This change makes sense to me. The Robot tests will closely resemble how the upgrade is actually executed by a user.

Reply:
Looks great. Feel free to push these commits in.

Reply:
Ok i just updated the branch with these commits. Waiting for CI